### DISCUSSION TITLE
[BUG]工具箱FTp  卸载不干净

### DISCUSSION THREAD

**Discussion by Alexzzz2008** [1Panel-dev/1Panel#5393] [Q&A]
### 1Panel 版本

1.10.10

### 问题描述

我不知道是什么原因   我安装FTP 后测试   一直报530错误  
索性卸载 FTP

执行命令  
sudo service pure-ftpd stop
sudo apt-get remove pure-ftpd
sudo rm -rf /etc/pure-ftpd/

再次进入 工具箱 FTP界面  

FTp 还是可以开启   新建用户名密码  报错
如图1和图2
![Sunny_20240609_230425](https://github.com/1Panel-dev/1Panel/assets/40814658/250c49fa-1f60-4372-a645-79c6b00ca191)

![Sunny_20240609_230450](https://github.com/1Panel-dev/1Panel/assets/40814658/edbff906-37f9-4879-9f16-5cca180105b8)

说明执行命令 卸载的FTP不干净,   不知道啥原因
正常情况 卸载后 不是应该 回到 初始状态  (提示你安装 pure-ftpd 吗)

### 重现步骤

1

**Comment by wanghe-fit2cloud** [comment]
1Panel 目前不负责 Pure-FTPd 的安装和卸载，只提供了一份可参考的安装教程。具体的安装和卸载方法请参阅 Pure-FTPd 官方文档。

**Comment by wawmb** [comment]
一样一样
